Makefile.js

```diff
@@ -708,6 +713,31 @@ target.checkLicenses = function() {
     });
 };
 
+target.checkGitCommit = function() {
+    var commitMsg = exec("git log -1 --format=format:%s --no-merges", { silent: true }).output;
+
+    echo("Validating Commit Message");
+
+    // If it is a release commit do not check any further
+    if (semver.valid(commitMsg)) {
+        return;
+    }
+
+    // Check for tag at start of message
+    if (!TAG_REGEX.test(commitMsg)) {
+        echo("Commit message must start with one of:\n  'Fix:'\n  'Update:'\n  'Breaking:'\n  'Docs:'\n  'Build:'\n  'New:'\n  'Upgrade:'");
+        exit(1);
+    }
```

Comment:
I think I should not exit here, but rather flag for failure so that all checks complete and all error messages are shown before exiting.  I will update tomorrow.